CIP6/CIP6.md

```diff
@@ -0,0 +1,885 @@
+
+## Motivation
+
+As the ecosystem around Cardano stake pools proliferate so will the desire to slice, organize and search pool information dynamically.  Currently the metadata referenced on chain provides 512 bytes that can be allocated across the four information categories ([delegation-design-specification Section 4.2)](https://hydra.iohk.io/build/790053/download/1/delegation_design_spec.pdf):
+
+| key           | Value                                |  Rules  |
+| ---           | ---                                  |  ---  |
+|  `ticker` | Pool ticker.  uppercase | 5 Characters Maximum, Uppercase letters and numbers |
+|  `description` | Pool Description.  Text that describes the pool | 50 Characters Maximum |
+|  `homepage` | A website URL for the pool  | 64 Characters Maximum, must be a valid URL |
+|  `name` | A name for the pool | 50 Characters Maximum |
+
+Many additional attributes can be envisioned for future wallets, pool explorers, and information aggregators.  The proposal below outlines an initial strategy for capturing this extended metadata.
+
+## Specification
+
+#### On Chain referenced metadata file
+We define one more line for the on chain referenced metadata file that references another json file on a url with the extended metadata.  The proposed metadata is as follows:
+
+| key           | Value                                | Rules  |
+| ---           | ---                                  | ---  |
+|  `ticker`       | Pool ticker.  uppercase              | 5 Characters Maximum, Uppercase letters and numbers  |
+|  `description` | Pool Description.  Text that describes the pool | 50 Characters Maximum |
+|  `homepage` | A website URL for the pool| 64 Characters Maximum, must be a valid URL |
+|  `name` | A name for the pool | 50 Characters Maximum |
+|  `extended` | A url for extended metadata| Optional, 64 Characters Maximum, must be a valid URL |
+
+#### Extended Metadata
+The file located at the URL for extended data is a json compliant text file with the following top level fields:
+
+| key           | Description                                |
+| ---           | ---                                  |
+| `serial` | set to YYYYMMDDxx on every update |
+|  `itn` | ITN pool validation data |
+|  `info` | Corporate image, contacts and about details |
+| `pools` | Pool details |
+
+Since this extended metadata file has no checksum and can be updated at any time by the pool operator, a **serial number** is useful to easily identify updates.
+
+The purpose of this first (**itn**) extension is the verifiable linking of an ITN pool ticker with its counterpart in Mainnet to identify fraudulent duplicates.
+
+#### Complete Extended Metadata Example
+```
+{
+    "serial": 2020072001,
+    "itn": {
+        "owner": "ed25519_pk1...",
+        "witness": "ed25519_sig1..."
+    },
+    "info": {
+        "CI": {
```

Comment:
What does "CI" stand for here?

Reply:
> What does "CI" stand for here?

https://en.wikipedia.org/wiki/Corporate_identity
mainly it is to make a bit more out of the all equal looking tickers.  Sort of a brand.
You can see some first attempts by tickers using smiley characters. imo not really professional looking but hey: trying to get some attention.
This also means such data (graphs, colours, ... ) must be used very carefully by the consuming applications, in order to not end up like a motley fairground. 
The whole extended metadata is more like an efficient and standardized way for SPOs to publish optional data, that can but mustn't be used by end-user applications.